src/internal/packager/git/common.go

```diff
@@ -23,6 +25,8 @@ const onlineRemoteName = "online-upstream"
 const offlineRemoteName = "offline-downstream"
 const onlineRemoteRefPrefix = "refs/remotes/" + onlineRemoteName + "/"
 
+var IsHash = regexp.MustCompile(`^[0-9a-f]{40}$`).MatchString
```

Comment:
lint error

Reply:
@jeff-mccoy 
What lint error are you seeing?

And how are you seeing it? I might not have something set up correctly b/c I'm not seeing it right now.

Reply:
![Screenshot 2023-02-01 at 5 47 22 PM](https://user-images.githubusercontent.com/882485/216193594-a2186524-e7d9-4fc1-a11d-c3ee8bf0cdc5.png)